app/views/sorns/search.html.erb

```diff
@@ -1,12 +1,18 @@
-<header id="search-header" class="margin-top-2">
-  <div class="grid-container">
+<header id="search-header" class="margin-top-2 home-intro">
+  <div class="grid-container site-intro">
     <div class="grid-row grid-gap grid-col-9">
-      <h1 id="site-title">SORN Dashboard</h1>
-      <p id="sub-title">This dashboard enables targeted search of System of Records Notices (SORN). The dashboard regularly pulls newly published SORNs from the Federal Register, and includes document from 1994 to the present day. <%= link_to "Learn more about this tool", about_path %></p>
+      <h1 class="site-title">SORN Dashboard</h1>
+      <p class="sub-title">This dashboard enables targeted search of System of Records Notices (SORN). The dashboard regularly pulls newly published SORNs from the Federal Register, and includes document from 1994 to the present day. <%= link_to "Learn more about this tool", about_path %></p>
     </div>
   </div>
 </header>
 
+
+<section class="grid-container">
+  <div class="grid-col-10 grid-offset-1 search-area margin-y-2">
```

Comment:
This hint text is a little "floaty" since it's not aligned to anything. I got it to align with the text box by changing it to `grid-offset-4` and adding `padding-left-2`